MovieApp/MyCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+
+import UIKit
+import PureLayout
+
+class MyCollectionViewCell: UICollectionViewCell {
+    
+    static let cellIdentifier = "crewCellIdentifier"
+    
+    private let crewMemberNameLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    private let crewMemberPositionLabel: UILabel = {
+        let label = UILabel()
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        contentView.addSubview(crewMemberNameLabel)
+        contentView.addSubview(crewMemberPositionLabel)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .top, withInset: 3)
+        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
+        crewMemberPositionLabel.autoPinEdge(.top, to: .bottom, of: crewMemberNameLabel, withOffset: 3)
+        crewMemberPositionLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
```

Comment:
Setting up constraints should not be done in `layoutSubviews` method. `layoutSubviews` method is called multiple times and because of that you will create multiple instances of the same constraint.

Move this to a separate function `defineLayout` or something like that and call it from the init, after you added your subviews to the view hierarchy. 

```
    override init(frame: CGRect) {
        super.init(frame: frame)
        contentView.addSubview(crewMemberNameLabel)
        contentView.addSubview(crewMemberPositionLabel)
        
        defineLayout()
    }

   func defineLayout() {
        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .top, withInset: 3)
        crewMemberNameLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
        crewMemberPositionLabel.autoPinEdge(.top, to: .bottom, of: crewMemberNameLabel, withOffset: 3)
        crewMemberPositionLabel.autoPinEdge(toSuperviewEdge: .leading, withInset: 0)
    }
```